config/routes.rb

```diff
@@ -34,5 +34,7 @@
   get 'errors/not_found'
   get 'errors/unauthorized'
 
+  get "/.well-known/acme-challenge/:id" => "letsencrypt#authenticate_key"
```

Comment:
Instead of creating a whole controller here, you can do something really simple directly in the routes.rb file:
```ruby
get "/.well-known/acme-challenge/:id", to: proc { |env| [200, {}, ["PVltCpMKHMFVvw2JnYNnsKovB8XNwbEm6iB_sjmTBvA.EOz8wpTt91fuDD0G0ziU0_tJHJNh11aZyXsVX3y7mro"]] }
```
I learned this in Obie Fernandez's “The Rails™ 4 Way".  👍